src/legal-officer/transaction-protection/LocContext.tsx

```diff
@@ -124,21 +138,105 @@ export interface Props {
 }
 
 export function LocContextProvider(props: Props) {
+
+    const UNKNOWN_FILE = "-";
     const { api } = useLogionChain();
-    const { openedLocRequests, closedLocRequests, axios } = useCommonContext();
+    const { axios } = useCommonContext();
     const [ contextValue, dispatch ] = useReducer(reducer, initialContextValue(props.locId));
+    const [ refreshing, setRefreshing ] = useState<boolean>(false);
+    const [ refreshCounter, setRefreshCounter ] = useState<number>(0);
```

Comment:
Isn't it possible to include `refreshing` and `refreshCounter` into `contextValue`?